mercury/resources.py

```diff
@@ -1,13 +1,61 @@
 from tastypie.resources import ModelResource
 from tastypie.authorization import Authorization
-from mercury.models import SimulatedData
+from mercury.models import (
+    TemperatureSensor,
+    AccelerationSensor,
+    WheelSpeedSensor,
+    SuspensionSensor,
+    FuelLevelSensor,
+)
 
+# TODO check with Dave,
 
-class SimulatedDataResource(ModelResource):
-    """This class exposes an API for the SimulatedData object with the resource name
+
+class TemperatureSensorResource(ModelResource):
+    """This class exposes an API for the TemperatureData object with the resource name
```

Comment:
`TemperatureData`? Also, I think since these comments for each class are 99% identical, we can just comment the module instead of each class.